devops/openshift/database/redis-ha-deploy.yml

```diff
@@ -148,6 +148,17 @@ objects:
               requests:
                 storage: ${PVC_SIZE}
           storageClassName: ${STORAGE_CLASS}
+  - apiVersion: policy/v1
+    kind: PodDisruptionBudget
+    metadata:
+      name: ${NAME}-pdb
+    spec:
+      selector:
+        matchLabels:
+          app: ${NAME}
+      minAvailable: 6
```

Comment:
Do we have a source of recommendation for Redis PDB configuration?

Reply:
As all the databases we use are statefulsets and I followed the documentation below
https://kubernetes.io/docs/tasks/run-application/configure-pdb/#identify-an-application-to-protect

The numbers I have put in there are values I have like maxUnavailable for each pods, when the disruption happens.

Reply:
Just wondering if there is a recommendation from Gov or if we can get something from RocketChat.